schema/message_fmt2/testgen_schema.json

```diff
@@ -209,13 +196,17 @@
       "type": "string"
     },
     "src": {
-      "description": "The MF2 syntax source.",
-      "type": "string"
-    },
-    "srcs": {
-      "description": "An array of strings to be concatenated to get the MF2 syntax source.",
-      "type": "array",
-      "items": { "type": "string" }
+      "oneOf": [
+        {
+          "description": "The MF2 syntax source.",
+          "type": "string"
+        },
+        {
+          "description": "The MF2 syntax source, as an array of strings to be concatenated.",
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      ]
```

Comment:
Nice!

Reply:
Yep, this is good change, thanks. @mihnita was asking for something like this when he went ahead and wrote his [ICU4J unit tests as such](https://github.com/unicode-org/icu/blob/main/icu4j/main/core/src/test/resources/com/ibm/icu/dev/test/message2/icu-test-functions.json) (modulo a different field name), given that JSON doesn't give any support to make multi-line strings developer-friendly